test/runAll.spec.js

```diff
@@ -35,6 +39,23 @@ describe('runAll', () => {
     return expect(runAll(scripts, getConfig({}))).resolves.toEqual('No tasks to run.')
   })
 
+  it('should resolve the promise with no files', () => {
+    expect.assertions(1)
+    return expect(
+      runAll(scripts, getConfig({ linters: { '*.js': ['echo "sample"'] } }))
+    ).resolves.toEqual({})
+  })
+
+  it('should not skip tasks if there are files', () => {
```

Comment:
I'm not sure what this test is asserting, TBH. Should it be looking at the console print and match it to a snapshot?